docs/mirror-bitbucket-cloud.md

```diff
@@ -0,0 +1,18 @@
+# Mirroring Bitbucket Cloud organizations and repos in Snyk
```

Comment:
could you link to this from the root docs as well?

Reply:
Done